### DISCUSSION TITLE
Newman parallel iterations

### DISCUSSION THREAD

**Discussion by sambhavjai** [postmanlabs/gsoc#30] [Ideas]
Hi 
This is Sambhav Jain, a final year student student at Netaji Subhas Institute of Technology. I have drafted a proposal for having parallel iterations in Newman and would request the Postman team to please look at this proposal for GSOC 2021 and provide me feedback for the same.
I have been trying to contact the mentors on discourse channel but I have failed to get any reply on the same.

(https://docs.google.com/document/d/1BDPmUALcYlVCUCkNY88bSK8IVVnu6AlE1iVNU-JuGP0/edit?usp=sharing)

**Comment by Sharath-Postman** [comment]
Thank you for creating thi. Mentors have responded to your community topic and are evaluating your proposal.
